leds/leds.ino

```diff
@@ -126,6 +131,12 @@ void doLeds() {
   Serial.print("entering doLeds with: ");
   Serial.print(lastReceived);
   Serial.println("");
+
+  // Not our id, message wasn't for us
+  if( !strstr(lastReceived, idString)) {
+    return;
+  }
```

Comment:
Can we instead use PubNub Stream Filters? Then you should be able to filter out any messages that are not for this device.

Reply:
This is a client feature, isn't it? I can't find any traces of stream filters in the pubnub arduino client.